controllers/propagator/replicatedpolicy_controller.go

```diff
@@ -97,13 +97,14 @@ func (r *ReplicatedPolicyReconciler) Reconcile(ctx context.Context, request ctrl
 			return reconcile.Result{}, nil
 		}
 
-		// do not handle a replicated policy which does not belong to the current cluster
```

Comment:
This change is handle replicated policies which belong to the current cluster?

Reply:
The special case being considered here, where it needs to skip dealing with a policy, is caused by the current cluster being a managed cluster for some other hub. In that hub-of-hubs situation, there can be a synced policy on this current cluster, which this reconciler would think is invalid because it can't find that root policy.